src/app/pages/StoryPage/StoryPage.jsx

```diff
@@ -238,14 +238,9 @@ const StoryPage = ({ pageData, mostReadEndpointOverride }) => {
     }
   `;
 
-  const MostReadSection = styled.section.attrs(() => ({
-    role: 'region',
-    'aria-labelledby': 'Most-Read',
-    'data-e2e': 'most-read',
-  }))``;
-
   const MostReadWrapper = ({ children }) => (
-    <MostReadSection>
+    // eslint-disable-next-line jsx-a11y/no-redundant-roles
```

Comment:
should we listen to this lint error?

Reply:
I've pruned instances of this comment and added `'jsx-a11y/no-redundant-roles': 'off',` to our ruleset.